scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -279,7 +260,7 @@ def build_graph(df, dataset):
     df = muon_calibration.define_corrected_muons(df, cvh_helper, jpsi_helper, args, dataset, smearing_helper, bias_helper)
 
     df = muon_selections.select_veto_muons(df, nMuons=1)
-    df = muon_selections.select_good_muons(df, 24, 100, dataset.group, nMuons=1, use_trackerMuons=args.trackerMuons, use_isolation=False)
+    df = muon_selections.select_good_muons(df, args.ptlmin, args.ptlmax, dataset.group, nMuons=1, use_trackerMuons=args.trackerMuons, use_isolation=False)
```

Comment:
I agree on having it in a separate PR unless it is required for other reasons here. 

About the changes in yields, we should make sure that our plotting scripts do not include overflow bins in projections by default, since those bins should in principle be empty (I assume you saw changes in some eta distributions when projecting pt). Or, we should ensure that we can control if those bins should be included in projections, if there are cases where we want to see their contributions.